src/amplitude-client.js

```diff
@@ -1140,7 +1140,7 @@ AmplitudeClient.prototype.identify = function (identify_obj, opt_callback) {
   }
 
   // if identify input is a proxied object created by the async loading snippet, convert it into an identify object
-  if (type(identify_obj) === 'object' && identify_obj.hasOwnProperty('_q')) {
+  if (Object.prototype.hasOwnProperty.call(identify_obj, '_q')) {
```

Comment:
trying to call this on a non-object-like will fail, so I got to remove a check here :)